src/main/java/bio/terra/service/auth/iam/IamAction.java

```diff
@@ -32,7 +32,6 @@ public enum IamAction {
   LINK_SNAPSHOT,
   UNLINK_SNAPSHOT,
   GET_SNAPSHOT_BUILDER_SETTINGS,
-  UPDATE_SNAPSHOT_BUILDER_SETTINGS,
```

Comment:
Thank you! (Just an extra reminder that SAM change needs to be merged first)

Reply:
https://github.com/broadinstitute/sam/pull/1439 !